packages/seo-store/src/analysis/slice/index.js

```diff
@@ -14,13 +14,16 @@ export const analysisSelectors = {
 	...resultsSelectors,
 	...configSelectors,
 	selectPaper: createSelector(
+		formSelectors.selectTitleTemplate,
+		formSelectors.selectDescriptionTemplate,
 		formSelectors.selectSeoTitle,
 		formSelectors.selectMetaDescription,
 		formSelectors.selectSlug,
 		editorSelectors.selectContent,
 		editorSelectors.selectPermalink,
 		editorSelectors.selectFormattedDate,
-		( seoTitle, metaDescription, slug, content, permalink, date ) => ( { seoTitle, metaDescription, slug, content, permalink, date } ),
+		( titleTemplate, descriptionTemplate, seoTitle, metaDescription, slug, content, permalink, date ) =>
+			( { titleTemplate, descriptionTemplate, seoTitle, metaDescription, slug, content, permalink, date } ),
 	),
```

Comment:
I am not entirely sure that this is the correct approach to add the title and description templates as part of the paper selector. But at this moment I cannot think of a better approach with this situation in mind:
* We don't want to save the SEO title and meta description to the store if they are exactly the same as the template.
* The analysis data stored in Paper are retrieved from the store. This means that, when the data for SEO title and description in the store are empty (even though we display the templates in the snippet editor), the assessments that use those two data will return with incorrect result.